packages/shared/styles/global.ts

```diff
@@ -5,5 +5,10 @@ export const injectGlobalCss = () => {
     .ReactVirtualized__Grid {
       outline: none;
     }
+
+    .ReactVirtualized__Grid:nth-child(1),
+    .BottomLeftGrid_ScrollWrapper {
+      box-shadow: 5px 0 5px -2px #d6d4d4;
```

Comment:
Can we move this color to a central place for colors? I think having an organisation like this should be one of the core elements of the ui-kit. I personally would also like to have the default shadow width and spread in variables, but I don't have hard feelings around this.

Reply:
@mperrotti what would you say?

Reply:
Definitely agree that we should have a collection of "design tokens" to pull from. For now, the design tokens can live here in DC/OS UI Kit, but in the future it would be cleaner to put them in their own package.

For now, we export a new mapping from `packages/shared/styles/shadow.ts`, that includes something like:
`shadowRight: 5px 0 5px -2px #d6d4d4`

Reply:
Agreed on adding the color as well. `shadow` is a little confusing as a variable name for a color though. We could benefit from namespacing colors